### PR TITLE
Ensure we have a <dt> before every <dd> for valid HTML

### PR DESCRIPTION
**Pull request by Camwyn**
Simple fix for the issue, might require a bit more conversation before it goes live.

[BTRIA-1847]

[BTRIA-1847]: https://theeventscalendar.atlassian.net/browse/BTRIA-1847?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Review by redscar — DISMISSED** [review]

**Review by bordoni — CHANGES_REQUESTED** [review]
Changelog, and update the docblock with the TBD version.

File: src/views/modules/meta/venue.php [hunk 1]
```diff
@@ -24,9 +24,11 @@
 	<dl>
 		<?php do_action( 'tribe_events_single_meta_venue_section_start' ) ?>
 
+		<dt style="display:none;"><?php // This element is just to make sure we have a valid HTML ?></dt>
```
Comment (Camwyn) [inline src/views/modules/meta/venue.php:27]: More accessible version of this?

File: src/views/modules/meta/organizer.php [hunk 2]
```diff
@@ -62,14 +62,8 @@
...
 				?>
-				<?php if ( ! empty( $website_title ) ): ?>
-					<dt class="tribe-organizer-url-label">
-						<?php echo esc_html( $website_title ) ?>
-					</dt>
-				<?php endif; ?>
-				<dd class="tribe-organizer-url">
-					<?php echo $website; ?>
-				</dd>
+				<dt class="tribe-organizer-url-label"><?php echo esc_html( $website_title ) ?></dt>
```
Comment (Camwyn) [inline src/views/modules/meta/organizer.php:65]: Make this more accessible if $website_title is empty?

File: src/views/modules/meta/venue.php (hunk 1 above)
Comment (Camwyn) [reply src/views/modules/meta/venue.php:27]:
```suggestion
		<dt class="tribe-common-a11y-visual-hide">Venue information</dt>
```

File: src/views/modules/meta/venue.php [hunk 3]
```diff
@@ -44,9 +46,7 @@
 		<?php endif ?>
 
 		<?php if ( ! empty( $website ) ): ?>
-			<?php if ( ! empty( $website_title ) ): ?>
-				<dt class="tribe-venue-url-label"> <?php echo esc_html( $website_title ) ?> </dt>
-			<?php endif ?>
+			<dt class="tribe-venue-url-label"> <?php echo esc_html( $website_title ) ?> </dt>
```
Comment (Camwyn) [inline src/views/modules/meta/venue.php:49]: same as above

File: src/views/modules/meta/venue.php [hunk 4]
```diff
@@ -24,9 +24,11 @@
 	<dl>
 		<?php do_action( 'tribe_events_single_meta_venue_section_start' ) ?>
 
+		<dt style="display:none;"><?php // This element is just to make sure we have a valid HTML ?></dt>
 		<dd class="tribe-venue"> <?php echo tribe_get_venue() ?> </dd>
 
 		<?php if ( tribe_address_exists() ) : ?>
+			<dt style="display:none;"><?php // This element is just to make sure we have a valid HTML ?></dt>
```
Comment (Camwyn) [inline src/views/modules/meta/venue.php:31]: same as above

**Comment by Camwyn** [comment]
This is covered by Eugene's PR https://github.com/the-events-calendar/the-events-calendar/pull/4329

**Pull request closed without merging**